hexrd/fitting/calibration.py

```diff
@@ -4,14 +4,14 @@
 
 from scipy.optimize import leastsq, least_squares
 
+from tqdm import tqdm
```

Comment:
@joelvbernier Should we add `tqdm` as a dependency in the `setup.py` and conda recipe?

Reply:
This was added in #401.